k8s/notify_stale_pending_prs.yml.j2

```diff
@@ -0,0 +1,22 @@
+replicaCount: 1
+image:
+  repository: ghcr.io/chia-network/github-bot
+  tag: {{ DOCKER_TAG }}
+
+deployment:
+  args:
+    - label-prs
```

Comment:
This action should be notify-pending or something right? Label-prs is my command

Reply:
Added the appropriate commands